tests/OutputFormatTest.php

```diff
@@ -280,6 +310,8 @@ public function ignoreExceptionsOff(): void
     {
         $this->expectException(OutputException::class);
 
+        $outputFormat = OutputFormat::create()->setIgnoreExceptions(false);
```

Comment:
This test uses two separate `OutputFormat`s.

Reply:
> it doesn't seem relevant to me to use two instances with identical settings

Me neither - given that it's just testing that an exception is thrown when attempting to render a declaration block without a selector.